src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
+// reducer
+const initialState = [{ status: 'UNDER CONSTRUCTION' }];
+
+const categoryReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return state[0].status;
```

Comment:
- [ ] Kindly make the `CHECK_STATUS` case return the string `Under construction` as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=For%20the%20check%20status%20action%20return%20a%20string%20%22Under%20construction%22.).
### Hint: 
```ruby
   case CHECK_STATUS:
       return 'Under construction';
```